src/cc_catalog_airflow/dags/provider_api_scripts/cleveland_museum_of_art.py

```diff
@@ -0,0 +1,177 @@
+            raise Exception('json response could not be captured')
+            response_json = None
+
+    if (
+        response_json is None or response_json.get('error') is not None
+        ):
+        logger.warning('Bad response_json')
+        logger.info('Retrying \n'
+                    f'endpoint -- {ENDPOINT} \t'
+                    f' with parameters -- {query_param} ')
+
+        _get_response(
+            query_param,
+            endpoint,
+            retries = retries - 1
+            )
+    
+def _response_handler(
+    batch
+    ):
+    for data in batch:
+        license_ = data.get('share_license_status','').lower()
+        
+        if license_ != 'cc0':
+            logger.error('Wrong license image')
+            continue
+        license_version = '1.0'
+
+        foreign_id = data.get('id')
+        foreign_landing_url = data.get('url',None)
+        image_data = data.get('images',None)
+        image_url,key = _get_image_type(image_data)
+
+        if image_url is not None:
+            width = image_data[key]['width']
+            height = image_data[key]['height']
+        
+        title = data.get('title',None)
+        metadata = _get_metadata(data)
+        
+        if data.get('creators'):
+            creator_name = data.get('creators')[0].get('description','')
+        else:
+            creator_name = ''
+
+        total_images = image_store.add_item(
+                        foreign_landing_url=foreign_landing_url,
+                        image_url=image_url,
+                        license_=license_,
+                        license_version=license_version,
+                        foreign_identifier=foreign_id,
+                        width=width,
+                        height=height,
+                        title=title,
+                        creator=creator_name,
+                        meta_data=metadata,
+                        source=PROVIDER
+                        )
+    
+    return total_images
+
+def _get_image_type(
+    image_data
+    ):
+    if image_data.get('web'):
+        key = 'web'
+        image_url = image_data.get('web').get('url',None)
+    
+    elif image_data.get('print'):
+        key = 'print'
+        image_url = image_data.get('print').get('url',None)
+
+    elif image_data.get('full'):
+        key = 'full'
+        image_url = image_data.get('full').get('url',None)
+    else:
+        image_url = None
+
+    if image_url is None:
+        key = None
+        
+    return image_url,key
+
+def _get_metadata(data):
+    metadata = {}
+
+    metadata['accession_number'] = data.get('accession_number','')
+    metadata['technique'] = data.get('technique','')
+    metadata['date'] = data.get('creation_date','')
+    metadata['credit_line'] = data.get('creditline','')
+    metadata['classification'] = data.get('type','')
+    metadata['tombstone'] = data.get('tombstone','')
+    metadata['culture'] = ','.join(list(data.get('culture','')))
```

Comment:
This prevents the script from running all the way through, since there can be JSON `null` values (which become python `Nonetype`s) in this list, and you can't join `Nonetype`s.
```suggestion
    metadata['culture'] = ','.join(
        [i for i in data.get('culture', []) if i is not None]
    )
```